microsoft-edge/progressive-web-apps-chromium/microsoft-store.md

```diff
@@ -95,19 +95,20 @@ Now that you have your app publishing information, generate a Windows app packag
 To generate an app package, complete the following actions.  
```

Comment:
(For visibility, adding same comment in the Files tab in addition to on the Conversation page.)
Justin/Maria, Can we vouch that this overall article is reasonably confirmed to be up-to-date? If so, please change the date at top to the current date.